pkg/logs/message/origin.go

```diff
@@ -21,8 +21,14 @@ type Origin struct {
 }
 
 // NewOrigin returns a new Origin
-func NewOrigin() *Origin {
-	return &Origin{}
+func NewOrigin(identifier string, logSource *config.LogSource, offset int64, timestamp string, tags []string) *Origin {
```

Comment:
Do we need all of these in a single constructor ? Can't we refine the class ? 
Currently it looks like a serious mixed bags of parameters..
eg:
```
+	return message.NewOrigin("", w.source, 0, "", nil)
```